lua/csharp/features/user-secrets/init.lua

```diff
@@ -0,0 +1,95 @@
+local M = {}
+local logger = require("csharp.log")
+local utils = require("csharp.utils")
+local dotnet_cli = require("csharp.modules.dotnet-cli")
+
+-- Creates the user secret file if the same
+-- doesn't exists
+local function _ensure_secret_exists(user_secret_file_path)
+  local file, _ = io.open(user_secret_file_path, "r+")
+
+  if not file then
+    os.execute("mkdir -p " .. user_secret_file_path)
+    os.execute("chmod 600 " .. user_secret_file_path)
```

Comment:
Why do we need to chmod the file? The current user should have enough permissions to read and write the file, even if the file was created via the cli `dotnet user-secrets`.

Reply:
That was a mistake that I made, I was thinking that I was having some issue related to permissions but It was something else unrelated.